### PR TITLE
12 css pirates: Journal week 3/Tutorial

### PR DESCRIPTION
**Pull request by jeremyrist**

### CODE REVIEW

**Comment by ShayHall** [comment]
Yo, good thorough work.  I think a little more "philosophy", as you call it, in the journal entry would be more what they're looking for, but the content in both is solid.  I really like the images in the tutorial.  :+1: :soccer:  If you could give me a review, that'd be great.  Thanks.  https://github.com/ShayHall/shayhall.github.io/pull/6

**Comment by karjac14** [comment]
Wow this draft can turn into a reference book, I know a printing house.  hehe. Great work for summarizing profoundly. :+1:

**Comment by ironyarder77** [comment]
Jeremy-This is good stuff. It seems like you are really on track pal. keep it up. I enjoy your writing style.
I like the even and balanced mix of technical notes and reflections of class.

File: journal-week-3.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+
+What did we do in week 3? 
+
+* Much like The Donald, I don't have time for political correctness, and I also don't have time for philosophical reflection on my current state of mind so here's some bullet points from week 3.
```
Comment (kellymurray) [inline journal-week-3.md:4]: :laughing:

File: tutorial-week-3.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Grid-Based Layout:
+
+* In this brief tutorial we will talk about the Grid-Based Layout. 
+
+* This term is meant to describe what you get when you use CSS to design a web page.
```
Comment (kellymurray) [inline tutorial-week-3.md:5]: Not ALWAYS...more like a design method that you can use CSS for!

**Pull request merged**